server/accesslog.go

```diff
@@ -210,6 +219,14 @@ func (a *AccessLogServer) handleHTTPLogs(msg *als.StreamAccessLogsMessage_HttpLo
 	return nil
 }
 
+func grpcMethod(grpcService string, operation string) string {
+	if grpcService == "" && operation == "" {
+		return ""
+	}
+	return grpcService + "." + operation
```

Comment:
You can avoid creating multiple temporary strings here-- consider something like:

```
return fmt.Sprintf("%s.%s", grpcService, operation)
```

There is also `strings.Builder`, but that's probably overkill here, where there's a small number of strings, and they're combined in a fixed way.

Reply:
Good point.  I have updated the code per your suggestion.